src/test/kotlin/com/exactpro/th2/common/schema/TestJsonConfiguration.kt

```diff
@@ -214,23 +229,59 @@ class TestJsonConfiguration {
             "pass"
         )
 
+        private val CRADLE_NON_CONFIDENTIAL_COMBO_CONF_JSON = loadConfJson("cradle_non_confidential_combo")
         private val CRADLE_NON_CONFIDENTIAL_CONF_JSON = loadConfJson("cradle_non_confidential")
         private val CRADLE_NON_CONFIDENTIAL_CONF = CradleNonConfidentialConfiguration(
-            888,
+            false,
             111,
             123,
             321,
-            false,
             5000,
-            1280001,
-            1280002
+            1280002,
         )
+        private val CASSANDRA_STORAGE_SETTINGS_JSON = loadConfJson("cassandra_storage_settings")
+        private val CASSANDRA_STORAGE_SETTINGS = CassandraStorageSettings().apply {
+//            networkTopologyStrategy = NetworkTopologyStrategyBuilder()
+//                .add("A", 3)
+//                .add("B", 3)
+//                .build()
+            timeout = 4999
+//            writeConsistencyLevel = ConsistencyLevel.THREE
+//            readConsistencyLevel = ConsistencyLevel.QUORUM
+            keyspace = "test-keyspace"
+            keyspaceReplicationFactor = 1
+            maxParallelQueries = 1
+            resultPageSize = 2
+            maxMessageBatchSize = 3
+            maxUncompressedMessageBatchSize = 5
+            maxTestEventBatchSize = 8
+            maxUncompressedTestEventSize = 13
+            sessionsCacheSize = 21
+            scopesCacheSize = 34
+            pageSessionsCacheSize = 55
+            pageScopesCacheSize = 89
+            sessionStatisticsCacheSize = 144
+            pageGroupsCacheSize = 233
+            groupsCacheSize = 377
+            eventBatchDurationCacheSize = 610
+            counterPersistenceInterval = 987
+            composingServiceThreads = 1597
+        }
 
         private val PROMETHEUS_CONF_JSON = loadConfJson("prometheus")
         private val PROMETHEUS_CONF = PrometheusConfiguration("123.3.3.3", 1234, false)
 
         init {
-            OBJECT_MAPPER.registerModule(KotlinModule())
+            OBJECT_MAPPER.registerModule(
+                KotlinModule.Builder()
+                    .withReflectionCacheSize(512)
+                    .configure(KotlinFeature.NullToEmptyCollection, false)
+                    .configure(KotlinFeature.NullToEmptyMap, false)
+                    .configure(KotlinFeature.NullIsSameAsDefault, false)
+                    .configure(KotlinFeature.SingletonSupport, false)
+                    .configure(KotlinFeature.StrictNullChecks, false)
+                    .build()
+            )
 
             OBJECT_MAPPER.registerModule(RoutingStrategyModule(OBJECT_MAPPER))
```

Comment:
I think we could make the MAPPER filed public in AbstractCommonFactory. And use the exact same mapper here when we test the configuration deseriailzation